src/features/metadata-instance-editor/MetadataInstanceConfirmDialog.js

```diff
@@ -15,20 +15,37 @@ type Props = {
     onConfirm: Function,
 };
 
-const MetadataInstanceConfirmDialog = ({ onCancel, onConfirm, confirmationMessage }: Props) => (
-    <div className="metadata-instance-confirm-cover">
-        <div className="metadata-instance-confim-container">
-            <p className="metadata-instance-confirm-text">{confirmationMessage}</p>
-            <div className="metadata-instance-confirm-buttons">
-                <Button data-resin-target="metadata-confirmcancel" onClick={onCancel} type="button">
-                    <FormattedMessage {...messages.metadataCancel} />
-                </Button>
-                <PrimaryButton data-resin-target="metadata-confirmremove" onClick={onConfirm} type="button">
-                    <FormattedMessage {...messages.customRemove} />
-                </PrimaryButton>
+const MetadataInstanceConfirmDialog = ({ onCancel, onConfirm, confirmationMessage }: Props) => {
+    const cancelButtonRef = React.useRef(null);
+
+    React.useEffect(() => {
+        if (cancelButtonRef.current) {
+            cancelButtonRef.current.btnElement.focus();
+        }
+    }, []);
+
+    return (
+        <div className="metadata-instance-confirm-cover">
+            <div className="metadata-instance-confim-container">
+                <div role="alert">
```

Comment:
Does the `role` need to be added here or can it be added to the topmost `div`? Seeing if we can do away with this extra `div` that was added here

Reply:
Yes. Done!